### ISSUE TITLE
add support for npm ci

### ISSUE THREAD

**Issue by jehy** [swarajban/npm-cache#100]
It would be cool to use npm ci for CI environments, it would make npm-cache even faster : )

**Comment by jehy** [comment]
I just found that this feature exists in https://github.com/team-supercharge/npm-cache fork.

It would be cool to add those changes to upstream.

**Comment by jehy** [comment]
I created a pull request which contains `npm ci` as well as many other good things:
https://github.com/swarajban/npm-cache/pull/103

**Comment by fortinmike** [comment]
I agree that this would be very nice (actually, at work we can't use npm-cache because we use npm ci for "reproducible" builds).

As an aside, IMHO it would help to provide separate pull requests for this and for the other "good things". It would most probably help the project maintainer review and potentially merge your PR(s) as individual things.

**Comment by jehy** [comment]
Project seems abandoned so it does not matter much anyway. We made our fork and maintain it.

**Comment by fortinmike** [comment]
Is your fork published on npm?

**Comment by jehy** [comment]
My fork is not published on npm, because publishing it on github is enough for installation. You can use it from
https://github.com/jehy/npm-cache
Or from https://github.com/onetwotrip/npm-cache (we use npm-cache for onetwotrip.com ci).
